tests/helpers.py

```diff
@@ -61,6 +65,45 @@ def open(self, *args, **kwargs):
         return super(CTFdTestClient, self).open(*args, **kwargs)
 
 
+class CTFtime:
+    @contextmanager
+    def set():
```

Comment:
Can you leave docstrings on each of these functions detailing what they do? Just a simple explanation is fine. Like "This context manager can be used to setup start and end dates for a test CTFd".